sdks/java/io/jms/pom.xml

```diff
@@ -59,6 +46,15 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
```

Comment:
Oh, you mean on the "main" `findbugs-filter.xml`. I thought about one specific in the `JmsIO`. OK, I update this way.